### PR TITLE
fix(storage-browser): add 'sort' variant to button; remove incorrect class

### PR DESCRIPTION
**Pull request by timngyn**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by hbuchel — DISMISSED** [review]

File: packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx [hunk 1]
```diff
@@ -260,7 +260,8 @@ export const UploadControls = (): JSX.Element => {
         >
           {key in LocationActionViewColumnSortMap ? (
             <ButtonElement
-              className={`${CLASS_BASE}__table__data__button`}
```
[the remark this answers is not in the document]
Comment (timngyn) [reply packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx:263]: yes I think there is one from `components/DataTable`. I can import and use that for now

File: packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx [hunk 2]
```diff
@@ -261,7 +262,7 @@ export const UploadControls = (): JSX.Element => {
           {key in LocationActionViewColumnSortMap ? (
             <ButtonElement
               variant="sort"
-              className={`${CLASS_BASE}__table__header__button`}
+              className={`${TABLE_HEADER_BUTTON_CLASS_NAME}`}
```
Comment (calebpollman) [inline packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx:265]: Extraneous template literal

File: packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx [hunk 3]
```diff
@@ -202,7 +203,7 @@ export const LocationDetailViewTable = (): JSX.Element | null => {
           {LocationDetailViewColumnSortMap[column.key] ? (
             <Button
               variant="sort"
-              className={`${TABLE_HEADER_CLASS}__button`}
+              className={`${TABLE_HEADER_BUTTON_CLASS_NAME}`}
```
Comment (calebpollman) [inline packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx:206]: Extraneous template literal

Comment (timngyn) [reply packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx:206]: 🤦 oops sorry about that, thank you

**Review by hbuchel — APPROVED** [review]

**Review by calebpollman — APPROVED** [review]

**Pull request merged**